### DISCUSSION TITLE
Top AND Bottom Support interfaces with Substitute Material Control

### DISCUSSION THREAD

**Discussion by BladerunnerxRC** [OrcaSlicer/OrcaSlicer#7097] [Ideas]
I envision having the same control on the bottom as the top of support structures, especially when choosing the interface material. 

Application and benefit: Support filament materials allow for the clean separation of the support where it is not possible to 'support on build plate only.'

Problems and Issues:
Use support filament where you don't want the primary filament used in supports to adhere to the model at the bottom. Currently, the only option is using the support material filament for the entire support, which is wasteful, adds a tremendous amount of time to printing from filament swaps, and becomes costly.

**Comment by brysonhyte** [comment]
This is a slightly different use case, but the UI options remain the same. Using mixed materials between the supports and the main body (e.g., PLA/PETG) impacts tool changes and increases print time. Provide the ability to specify a different material for the Support Raft/Interfaces—both top and bottom—as well as the support body. This would help minimize tool changes by allowing the same material to be used for both the support and main body, ultimately reducing print time.

Another important consideration is that when printing supports with an alternate material, the Support Object X/Y distance may need to be adjusted based on the material. For example, when using PLA/PETG for supports, the support settings typically have a "zero" distance since they don’t bond. However, if you choose to use the same material for both the support body and the main body to reduce tool changes and print time, the offset distance may be too close, causing the materials to bond unintentionally.
